### DISCUSSION TITLE
assign individual names for several bms

### DISCUSSION THREAD

**Discussion by centro-max** [Louisvdw/dbus-serialbattery#237] [Ideas]
I have 4 batteries connected in parallel with 4 daly bms and would like to give each BMS its own name for easy differentiation. would it be possible?

![Screenshot_2022-10-04-20-52-38-744_tk klurige fullscreenbrowser](https://user-images.githubusercontent.com/93882970/193903913-b24d8580-a9f9-4ed9-b51f-35fd5bdf2209.jpg)

**Comment by Louisvdw** [comment]
Yes, we have a ticket for that. https://github.com/Louisvdw/dbus-serialbattery/issues/100
In fact it is already implemented and will be in the next beta.
![image](https://user-images.githubusercontent.com/26077665/193908718-95de1771-7f82-4410-b54d-eb89dc7ea3ff.png)

**Comment by centro-max** [comment]
Great 👍
